polling_stations/apps/file_uploads/views.py

```diff
@@ -0,0 +1,137 @@
+import json
+from datetime import datetime
+
+from django.conf import settings
+from django.contrib.admin.views.decorators import staff_member_required
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.db.models import Q
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_protect, ensure_csrf_cookie
+from django.utils.decorators import method_decorator
+from django.views.generic import TemplateView
+
+import boto3
+from boto.pyami.config import Config
+from botocore.exceptions import ClientError
+from marshmallow import Schema, fields, validate
+from marshmallow import ValidationError as MarshmallowValidationError
+
+from councils.models import Council
+
+
+MAX_FILE_SIZE = 150000000  # 150Mb
+S3_BUCKET_NAME = "pollingstations-data-dev"
+
+
+class FileSchema(Schema):
```

Comment:
Fine for MVP, but I reckon `FileSchema`, `UploadRequestSchema` and `get_s3_client` could be moved out of the view file at some point.